website/client/config/storybook/sprites.scss

```diff
@@ -0,0 +1,3 @@
+@for $i from 0 through 29 {
```

Comment:
We're up to 30 now 😩  I should probably implement something like this in app.vue, though.

Reply:
I can add it in app itself then always the same file used by both app and storybook

Reply:
That sounds good!

Reply:
should the way it is in the app.vue still be in separated <style>-Tags? or using it like that it would be combined into one ? would this "break" something? like was it separated into css files needed?

I know the image file itself that those are split up that was needed but css files ?

Reply:
on the other side - now the sprites.css file is only ONE request instead of 31 (added the /sprites.css into it as well) which should speed things up ... I think :D